packages/router/router.coffee

```diff
@@ -39,3 +39,22 @@ FlowRouter.route '/organizations/:_id',
     FlowLayout.render 'layout',
       main: 'organizationDetail'
       params: {"organizationId": params._id}
+
+FlowRouter.route '/myDatasets',
```

Comment:
Is there a reason you don't want this route to be "/datasets"? I think that's the more conventional place to put this.